modules/social_features/social_landing_page/social_landing_page.install

```diff
@@ -657,3 +658,12 @@ function social_landing_page_update_8808() {
     \Drupal::service('module_installer')->install(['social_book_featured']);
   }
 }
+
+/**
+ * Allow using any URLs in link fields by CM+.
+ */
+function social_landing_page_update_8809() {
+  foreach (['contentmanager', 'sitemanager'] as $role) {
+    user_role_grant_permissions($role, ['link to any page']);
```

Comment:
Hi @OH17L 
After checking this in our latest update I do notice some implications especially for our Tier 0+ customers.
This permission allows CM/SM to see menu links which they don't have access to. 
The menu link also gives access denied once visited.

<img width="773" alt="Screenshot 2020-10-06 at 15 51 41" src="https://user-images.githubusercontent.com/16667281/95210653-04415980-07ec-11eb-97c0-8eacc69c075c.png">

In this screenshot you see what it looks like for SM when this permission is disabled:
<img width="1579" alt="Screenshot 2020-10-06 at 15 48 34" src="https://user-images.githubusercontent.com/16667281/95210701-10c5b200-07ec-11eb-8847-ea9002cb221a.png">

Reply:
I created a task for this [here](https://getopensocial.atlassian.net/browse/YANG-3924).